bsip-recurringmonthly.md

```diff
@@ -0,0 +1,153 @@
+    BSIP: <BSIP number>
+    Title: Recurring Withdrawals Defined in Monthly Intervals
+    Authors: Taconator
+    Status: Draft
+    Type: Protocol
+    Created: 2018-08-15
+    Discussion: https://github.com/bitshares/bitshares-core/issues/1208
+
+# Abstract
+Recurring withdrawals can currently be defined in BitShares core with respect to any periodic interval greater than the minimum block interval (currently 3 seconds). However those intervals are not intuitive for many use cases such as intervals defined in terms of Gregorian calendar months. This BitShares Improvement Proposal describes describes possible defintions for the periodic intervals and possible implementations such that they are compatible with the existing recurring withdrawal capability. The adoption of of this proposal might increase the goods and services that are accessible through BitShares by making it easier for merchants to accept payment on an intuitive montly basis.
```

Comment:
"might increase" -> "will increase"

monthly is simply what the old economy uses